### PR TITLE
Re-adds fixture data for local development.

### LINKED ISSUE #2464: Create fixture data that can be used in localhost development
The Fedora based fixtures were removed in https://github.com/sul-dlss/argo/pull/2334

This means if you spin up localhost Argo for development purposes, you will have no data to view and thus no facet views either.  It would be useful to figure out how to get some basic data again for development purposes via fixtures, factories or some other mechanism.

### PR DESCRIPTION
**Pull request by justinlittman**
closes #2464

## Why was this change made?
Fixtures are helpful for local development.

Note that this does not re-add the fixtures for tests or CI.

## How was this change tested?
Local

## Which documentation and/or configurations were updated?
README

### CODE REVIEW

File: lib/tasks/argo_testing.rake [hunk 1]
```diff
@@ -23,3 +32,56 @@ task(:default).clear
...
+
+        handler = proc do |e, _attempt_number, _total_delay|
+          puts warn "ERROR loading #{file}:\n#{e.message}\n#{e.backtrace.join "\n"}"
+          errors << file
+        end
+        pid = "druid:#{File.basename(file, '.xml')}"
+        with_retries(max_tries: 3, handler: handler, rescue: [StandardError]) do |attempt|
+          puts "** File #{i}, Try #{attempt} ** file: #{file}"
+
+          ActiveFedora::FixtureLoader.import_to_fedora(file, pid)
```
Comment (jcoyne) [inline lib/tasks/argo_testing.rake:77]: I want to remove this dependency next week

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: lib/tasks/argo_testing.rake (hunk 1 above)
Comment (justinlittman) [reply lib/tasks/argo_testing.rake:77]: with_retries?

Comment (justinlittman) [reply lib/tasks/argo_testing.rake:77]: Can we take it out now?

Comment (jcoyne) [reply lib/tasks/argo_testing.rake:77]: ActiveFedora will be removed next week.  Maybe this can go in dor-services-app.

**Comment by jcoyne** [comment]
@justinlittman can this be closed now?

**Comment by justinlittman** [comment]
Is there another solution?

**Comment by jcoyne** [comment]
@justinlittman move into DSA?

**Pull request closed without merging**